tools/pipelines/test-perf-benchmarks.yml

```
@@ -70,160 +85,311 @@ stages:
         variables:
           releaseBuildVar: $[variables.releaseBuild]
           toolAbsolutePath: $(Build.SourcesDirectory)/tools/telemetry-generator
+          artifactPipeline: Build - client packages
+          ${{ if startsWith(variables['Build.SourceBranch'], 'refs/heads/test/') }}:
+            feed: https://pkgs.dev.azure.com/fluidframework/internal/_packaging/internal/npm/registry/
+          ${{ else }}:
+            feed: https://pkgs.dev.azure.com/fluidframework/internal/_packaging/build/npm/registry/
+
         steps:
         # Setup
         - checkout: self
           clean: true
```

I think I'd add a 
```suggestion
        # Setup. Need to checkout the repo in order to run @fluid-tools/telemetry-generator which we don't publish right now.
        - checkout: self
          clean: true
```

Small documentation bit. This way it'll be immediately clear why we still need to check out the repo even though the whole point of these changes is not needing to rebuild it in this pipeline.